packages/components/src/__future__/Tag/Tag/_docs/Tag-migration-guide.stories.tsx

```diff
@@ -23,106 +22,18 @@ const meta = {
 } satisfies Meta<typeof Tag>
 
 export default meta
+type Story = StoryObj<typeof meta>
 
-/** * This is a stand-in component for the legacy Tag's bake in LiveIcon - we should consider adding this as an actual component or replacing it */
-const LiveIconComponent = (): JSX.Element => (
-  <span className={styles.liveIcon}>
-    <LiveIcon
-      role="presentation"
-      classNameOverride={styles.liveIcon_base}
-      width="16"
-      height="16"
-      viewBox="0 0 16 16"
-      fill="none"
-    />
-    <LiveIcon
-      role="presentation"
-      classNameOverride={styles.liveIcon_1}
-      width="16"
-      height="16"
-      viewBox="0 0 16 16"
-      fill="none"
-    />
-    <LiveIcon
-      role="presentation"
-      classNameOverride={styles.liveIcon_2}
-      width="16"
-      height="16"
-      viewBox="0 0 16 16"
-      fill="none"
-    />
-    <LiveIcon
-      role="presentation"
-      classNameOverride={styles.liveIcon_3}
-      width="16"
-      height="16"
-      viewBox="0 0 16 16"
-      fill="none"
-    />
-  </span>
-)
-
-export const LiveIconComponentStory: StoryObj = {
```

Comment:
Looking at the migration guide, I think you'll want to reinstate this story for the code snippet.

A follow up task would be to speak with our designer Jess to check if we'd actually want people moving forward to just be using the static Future Icon `<Icon name="sensors" isPresentational />` instead, then updating the migration guide again with the decision.